.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["react", "es2015", "stage-0", "flow"],
+  "plugins": ["transform-decorators-legacy", "transform-object-rest-spread"]
+}
```

Comment:
A config auto add 1 newline at the end of file nhé a

Reply:
done!